elements/pfe-sass/extends/_extends.scss

```diff
@@ -56,3 +56,84 @@
     color: pfe-fetch(text) !important;
   }
 }
+
+%ordered-list-primary {
+  // size of the font inside of the circle
+  --FontSize: var(--pfe-list-FontSize, 24px);
+  // size of the space between the circle and the content
+  --gap: var(--pfe-list--gap, 24px);
+  // amount of space between list items
+  --margin: var(--pfe-list--margin, 24px);
```

Comment:
Why lowercase margin here and uppercase below?  Also should we call these out as MarginLeft etc.

Reply:
I replaced this with just using --pfe-list--gap.  They should be tied together by default.